onnxruntime/test/python/onnx_test_custom_ops.py

```diff
@@ -0,0 +1,107 @@
+import onnx
```

Comment:
@pranavsharma do you think onnxruntime/test/testdata is a good location for the test data from this file? If you think we shouldn't add test data inside onnxruntime or maybe there is a better location for it, please let me know. Thanks!